org.mixedrealitytoolkit.input/Subsystems/Hands/HandsProvider.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using MixedReality.Toolkit.Subsystems;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.Scripting;
+using UnityEngine.XR;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
```

Comment:
Can you please add the second sentence to the "remarks" tag.   On learn.microsoft.com, the summary bit is shown in a table alongside all other types in the assembly.  So having a shorter summary is preferred in that view.

```
    /// <summary>
    /// Provides a generic, base hands provider for subsystem use.
    /// </summary>
    /// <remarks>
    /// Extends the <see cref="MixedReality.Toolkit.Subsystems.HandsSubsystem.Provider">Provider</see> class and
    /// obtains hand joint poses from a <see cref="MixedReality.Toolkit.Input.HandDataContainer">HandDataContainer</see> class.
    /// </remarks>
```

Reply:
I made this change and will note it for the future, but I didn't think `internal` classes ended up on learn.microsoft.com.

Reply:
Doh it's internal...sorry about that comment.